src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
 
+        // This represents how much EC to consume per executed instruction.
+        // This would be the "variable" component of the processor's power.
+        // TODO: This value needs to be changed to a non-zero number and balanced.
```

Comment:
While it's true that we need to fiddle with the number to find the right balance, the placeholder we use in the meantime should probably be something other than just `0`, so we can see it at least having an effect of some sort and be able to test that the code is using it properly.